server/server.go

```diff
@@ -128,6 +131,10 @@ func NewServer(ctx *Context, stopper *util.Stopper) (*Server, error) {
 			return nil, err
 		}
 	}
+
+	//
+	s.sqlDB = sqlserver.NewDBServer(s.db)
```

Comment:
But we don't want packages with the names `server` and `wire`. Those are too general. We could keep the `sqlserver` and `sqlwire` package names while using the `sql/server` and `sql/wire` directories. Or we could get rid of `sql` and move the `sqlserver` and `sqlwire` packages into the top-level.